.github/CHANGELOG.md

```diff
@@ -125,7 +159,7 @@ J. Eli Bourassa, Guillaume Dauphinais, Ish Dhand, Theodor Isacsson, Josh Izaac,
 Leonhard Neuhaus, Nicolás Quesada, Aaron Robertson, Krishna Kumar Sabapathy, 
 Jeremy Swinarton, Antal Száva, Ilan Tzitrin.
```

Comment:
```suggestion
Theodor Isacsson, Leonhard Neuhaus, Nicolás Quesada, Aaron Robertson, Krishna Kumar Sabapathy, 
Jeremy Swinarton, Antal Száva, Ilan Tzitrin.
```

From #548 and #547.

Reply:
@thisac you're in the line above already :) GitHub hides it by default, since it hasn't had any changes:
![image](https://user-images.githubusercontent.com/2959003/115578899-42b9e400-a2f8-11eb-936c-20f137b17fe2.png)

Reply:
🤦